SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -251,6 +239,43 @@ public void run() {
         return (T) this;
     }
 
+    private void setDefaultMessageListener() {
+        setMessageListener(new ISwrveMessageListener() {
+            public void onMessage(final SwrveMessage message) {
+                // Start a Conversation activity to display the campaign
+                if (SwrveBase.this.context != null) {
+                    final Context ctx = SwrveBase.this.context.get();
+                    if (ctx == null) {
+                        SwrveLogger.e(LOG_TAG, "Can't display a in-app message without a context");
+                        return;
+                    }
+
+                    Intent intent = new Intent(ctx, SwrveInAppMessageActivity.class);
+                    intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                    intent.putExtra("inAppMessage", message);
+                    intent.putExtra("hideToolbar", config.isHideToolbar());
+                    intent.putExtra("minSampleSize", config.getMinSampleSize());
+                    intent.putExtra("defaultBackgroundColor", config.getDefaultBackgroundColor());
```

Comment:
Use constants for all those values: inAppMessage, hideToolbar, minSampleSize, defaultBackgroundColor